doc/tutorials/acrn_on_qemu.rst

```diff
@@ -11,7 +11,7 @@ with basic functionality such as running Service VM (SOS) and User VM (UOS) for
 
 This setup was tested with the following configuration,
 
-- ACRN Hypervisor: tag acrn-2020w19.5-140000p (Commit ID: b0ae9cfa2b374d3faad9f890ff61e88f97205ac8)
+- ACRN Hypervisor: tag v2.0 (Not support in later version)
 - ACRN Kernel: 5.4.28 (Commit ID: a8cd22f49f0b2b56e526150fe0aaa9118edfcede)
```

Comment:
That kernel is older is older than the `v2.0` kernel by about a couple of months. Can you confirm which was **really** used for the validation?

Reply:
Updated.